integration-tests/test-harness/src/EmbraceReactNativeTestHarness.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import {LogTestingScreen} from "./screens/LogTestingScreen";
+import {TracerProviderTestingScreen} from "./screens/TracerProviderTestingScreen";
+import {useEmbraceNativeTracerProvider} from "@embrace-io/react-native-tracer-provider";
+import {
+  NavigationContainer,
+  useNavigationContainerRef,
+} from "@react-navigation/native";
+import {createBottomTabNavigator} from "@react-navigation/bottom-tabs";
+import {NavigationTracker} from "@opentelemetry/instrumentation-react-native-navigation";
+
+const Tab = createBottomTabNavigator();
+
+export const EmbraceReactNativeTestHarness = () => {
+  const navigationContainerRef = useNavigationContainerRef();
+  const {tracerProvider} = useEmbraceNativeTracerProvider({});
+
+  return (
+    <NavigationContainer ref={navigationContainerRef}>
```

Comment:
@facostaembrace this is the setup I had where the navigation instrumentation was warning "Navigation ref is not available. Make sure this is properly configured. [Component Stack]", compare to the Expo setup that was working in `integration-tests/test-harness/src/EmbraceExpoTestHarness.tsx` above

I have an app built from this where the issue is reproducible or you can build a new one locally from the template

Reply:
will take a look today